src/components/CaseStudyTile/CaseStudyTile.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styles from './CaseStudy.module.css'
+
+interface CaseStudyTileProps{
+  projects : {
+    title?: string
+    subtitle: string
+    projectType : string
+    imageUrl?: string
+    direction: string
```

Comment:
Make direction:
`direction: 'left' | 'right';`

Because those are the only values that you can give

Reply:
## Fix attempts:
![image](https://github.com/rijinmk/vikas-portfolio/assets/110288123/305fa068-ad3e-4cd9-bb20-26262d39af04)

![image](https://github.com/rijinmk/vikas-portfolio/assets/110288123/51d88068-6da4-42d3-abd3-e421c7c878db)


## Linter Error for both fix attempts: 
![image](https://github.com/rijinmk/vikas-portfolio/assets/110288123/7de9a099-d715-4a82-b155-11346ae84c4e)